### PR TITLE
[RU] advanced.error-handling -> 1.1.0

### PR DESCRIPTION
**Pull request by IgorPolyakov**

### CODE REVIEW

File: ru/lessons/advanced/error-handling.md [hunk 1]
```diff
@@ -11,6 +11,17 @@ title: Обработка ошибок
 
 {% include toc.html %}
 
+## Общие соглашения
+
+На данный момент сообщество Elixir пришло к некоторым соглашениям относительно возврата ошибок:
+
+* Для ошибок, которые являются частью обычной работы функции (например, пользователь ввел неверный тип даты), функция возвращает соответственно `{: ok, result}` и `{: error, cause}`.
+* Для ошибок, которые не являются частью обычных операций (например, невозможность проанализировать данные конфигурации), вы генерируете исключение.
```
Comment (brain-geek) [inline ru/lessons/advanced/error-handling.md:19]: I'd suggest dropping `данные` word here

File: ru/lessons/advanced/error-handling.md [hunk 2]
```diff
@@ -11,6 +11,17 @@ title: Обработка ошибок
...
+
+Обычно мы обрабатываем стандартные ошибки потока с помощью [сопоставления с образцом](../../basics/pattern-matching/), но в этом уроке мы сосредоточимся на втором случае - на исключениях.
+
+Часто в общедоступных API вы также можете найти вторую версию функции с расширением! (`example!/1`), который возвращает развернутый результат или вызывает ошибку.
```
Comment (brain-geek) [inline ru/lessons/advanced/error-handling.md:23]: `вторую` is word-to-word translation, IMO it would sound a bit better with `альтернативную`.

**Comment by brain-geek** [comment]
@IgorPolyakov Thanks for the contributions, everything looks awesome! However, I found two places which could be discussed.

**Comment by IgorPolyakov** [comment]
@brain-geek Tthanks for the feedback!) I fixed two remarks, I think it looks better now)

**Review by brain-geek — APPROVED** [review]

**Comment by brain-geek** [comment]
@IgorPolyakov Thanks for the contribution!

**Pull request merged**